### ISSUE TITLE
Normalize with `sklearn.preprocessing.StandardScaler` and preprocess with `tf.keras.preprocessing.sequence.TimeseriesGenerator`

### ISSUE THREAD

**Issue by amyshenin-tech** [amyshenin-tech/stock-market-gan#1]
- https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.StandardScaler.html
- https://www.tensorflow.org/api_docs/python/tf/keras/preprocessing/sequence/TimeseriesGenerator
- https://machinelearningmastery.com/how-to-use-the-timeseriesgenerator-for-time-series-forecasting-in-keras/
- https://github.com/grudloff/stock_market_GAN/blob/master/TimeSeries_GAN.ipynb
